### PR TITLE
Debot approve

### PR DESCRIPTION
**Pull request by Keshoid**
Add `approve` callback to ask  from browser permissions to perform DeBot oeprations.

### CODE REVIEW

File: ton_client/src/json_interface/debot.rs [hunk 1]
```diff
@@ -33,6 +34,9 @@ pub enum ResultOfAppDebotBrowser {
     },
     /// Result of debot invoking.
     InvokeDebot,
+    Approve {
```
Comment (AlexeyVavilin) [inline ton_client/src/json_interface/debot.rs:37]: Add comment

File: ton_client/src/debot/calltype.rs [hunk 2]
```diff
@@ -308,21 +331,75 @@ fn build_answer_msg(
...
+) -> ClientResult<DebotActivity> {
+    let result = run_executor(
+        client.clone(),
+        ParamsOfRunExecutor {
+            message: msg.clone(),
+            account: AccountForExecutor::Account {
+                boc: target_state,
+                unlimited_balance: None,
+            },
+            execution_options: None,
```
Comment (AlexeyVavilin) [inline ton_client/src/debot/calltype.rs:371]: Better use `..Default::default()` instead of `None`s

**Review by AlexeyVavilin — CHANGES_REQUESTED** [review]

File: ton_client/src/json_interface/debot.rs (hunk 1 above)
Comment (Keshoid) [reply ton_client/src/json_interface/debot.rs:38]: Fixed

File: ton_client/src/json_interface/debot.rs [hunk 3]
```diff
@@ -79,7 +83,10 @@ pub enum ParamsOfAppDebotBrowser {
         /// Internal message to DInterface address. Message body contains
         /// interface function and parameters.
         message: String,
-    }
+    },
```
[the remark this answers is not in the document]
Comment (Keshoid) [reply ton_client/src/json_interface/debot.rs:88]: Fixed

**Review by AlexeyVavilin — APPROVED** [review]

**Pull request merged**